Pipfile

```diff
@@ -13,6 +13,7 @@ flask-cors = "*"
 webargs = "*"
 flask-socketio = "*"
 eventlet = "*"
+py2neo = "*"
```

Comment:
py2? Is that alright? :D

Reply:
py2neo is the python library for neo4j doesn't have anything to do with python 2 or python 3